spec/wings/model_transformer_spec.rb

```diff
@@ -172,7 +172,7 @@
           .to have_attributes title: book.title,
                               contributor: book.contributor,
                               description: book.description
-        expect_ids_to_match(subject.build.page_ids, ['pg1', 'pg2'])
+        expect(subject.build.page_ids).to match_valkyrie_ids_with_active_fedora_ids(['pg1', 'pg2'])
```

Comment:
Moved to be custom matcher in `wings/support/matchers` so it can be used across spec files.